identity/identity_verification.go

```diff
@@ -25,7 +25,7 @@ const (
 // VerifiableAddressType must not exceed 16 characters as that is the limitation in the SQL Schema
 //
 // swagger:model identityVerifiableAddressType
-type VerifiableAddressType string
+type VerifiableAddressType = string
```

Comment:
Can we remove this type completely? Or should we keep it for documentation purposes? If yes, I'd suggest to revert the changes where you replace this type by string in other places.

Reply:
It's still used in a lot of places. I think keeping it doesn't hurt.